scenarios/kubernetes_kops_aws.py

```diff
@@ -165,6 +165,8 @@ def main(args):
                (args.cluster, zones, args.state, args.nodes, aws_ssh))
     if args.image:
         e2e_opt += ' --kops-image=%s' % args.image
+    if args.kops_args:
+        e2e_opt += ' --kops-args=\"%s\"' % args.kops_args
```

Comment:
`cmd` here is "docker run", not kubetest, though, so if I do that docker thinks I'm passing in a repo/tag

I want to believe this is why the other kubetest args (--kops-cluster) are forced to be passed in via E2E_OPT, we have to pipe it all the way through the python script -> docker run -> e2e-runner.sh -> kubetest.